src/code/z_scene.c

```diff
@@ -196,19 +195,16 @@ void func_80098508(GlobalContext* globalCtx, SceneCmd* cmd) {
     Object_Spawn(&globalCtx->objectCtx, linkObjectId);
 }
 
-// Scene Command 0x01: Actor List
-void func_800985DC(GlobalContext* globalCtx, SceneCmd* cmd) {
+void Scene_CommandActorList(GlobalContext* globalCtx, SceneCmd* cmd) {
     globalCtx->numSetupActors = cmd->actorList.num;
     globalCtx->setupActorList = SEGMENTED_TO_VIRTUAL(cmd->actorList.segment);
 }
 
-// Scene Command 0x02: Unused 02
-void func_80098630(GlobalContext* globalCtx, SceneCmd* cmd) {
+void Scene_CommandUnused2(GlobalContext* globalCtx, SceneCmd* cmd) {
     globalCtx->unk_11DFC = SEGMENTED_TO_VIRTUAL(cmd->unused02.segment);
```

Comment:
I'm not sure how much we want to infer from MM, but `scene.c` from OoT and MM seem to mirror each other quite closely for all commands <= 0x18, and here this command gets used for camera data related to cutscenes (which very closely mimics bgCamData for reading camera data from bg entities). Is it a stretch to assume that was the intent but they didn't have enough time to implement it? Or would it be preferred to leave it as unknown. Just thought I'd put that context out there.

Reply:
it definitely is the same global context field, given its placement relative to other things. Id say its fine to name it the same as mm with a comment explaining that its unused but inferred from mm

Reply:
I agree with fig, but actually I'm wondering if it makes sense to backport the name since there's in OoT no trace of the thing this command implements in MM, there's nothing in OoT equivalent to MM's `CsCamData`, so I would prefer leaving it as unk.

I can still comment on it being used for a new engine feature in MM, would that be enough?

https://github.com/zeldaret/mm/blob/0ccd71c2da5c078878b6428e40952ab221ff8ead/include/z64.h#L1349
```c
    /* 0x18858 */ CsCamData* csCamData;
```

https://github.com/zeldaret/mm/blob/0ccd71c2da5c078878b6428e40952ab221ff8ead/src/code/z_scene.c#L181
```c
// SceneTableEntry Header Command 0x02: List of cameras for actor cutscenes
void Scene_HeaderCmdActorCutsceneCamList(GlobalContext* globalCtx, SceneCmd* cmd) {
    globalCtx->csCamData = Lib_SegmentedToVirtual(cmd->csCameraList.segment);
}
```

Reply:
I would personally name things the same and leave it as void*, with comments on both the member and the function that the names came from mm and it doesnt functionally work here and that the name comes from mm

we know for sure that its a pointer/segmented address from the oot perspective cause of `SEGMENTED_TO_VIRTUAL`, which is why I think void* is fair

Reply:
Why port the name though if it was only later established what to use it for?

Reply:
helps tie the codebases together, and leaving things as unk sucks when we know what it was intended for

Reply:
dunno, can see how others feel i guess
The function name would certainly be weird to have cause the concept of Actor Cutscenes dont exist, but I think its still fair to name the global context variable something

Reply:
yea i do see your point, and dont think its anything that needs to be addressed right away

it can stay as unk for now, but i would prefer if the global context field was named after it being related to the scene stuff in some way, as elliptic originally suggested